src/main/scala/mesosphere/marathon/event/Events.scala

```diff
@@ -206,6 +207,16 @@ case class AppTerminatedEvent(
   eventType: String = "app_terminated_event",
   timestamp: String = Timestamp.now().toString) extends MarathonEvent
 
+case class UnhealthyTaskKillEvent(
+  appId: PathId,
+  taskId: Task.Id,
+  version: Timestamp,
+  health: Health,
```

Comment:
I thougt it will be nice to have more details with event. I'll reduce it to keep only `lastFailureCause`